host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -651,70 +622,30 @@ defmodule HostCore.Actors.ActorModule do
 
     Logger.info("Starting actor #{claims.public_key}")
 
-    ClaimsManager.put_claims(host_id, lattice_prefix, claims)
-    ActorRpcSupervisor.start_or_reuse_consumer_supervisor(lattice_prefix, claims)
-
-    {:ok, agent} =
-      Agent.start_link(fn ->
-        %State{
-          claims: claims,
-          instance_id: UUID.uuid4(),
-          healthy: false,
-          annotations: annotations,
-          lattice_prefix: lattice_prefix,
-          host_id: host_id
-        }
-      end)
-
-    imports = %{
-      wapc: Imports.wapc_imports(agent),
-      wasmbus: Imports.wasmbus_imports(agent)
-    }
-
-    # we consider a hash of bytes as a unique key
-    key = :sha256 |> :crypto.hash(bytes) |> Base.encode16()
-
-    module =
-      case :ets.lookup(:module_cache, key) do
-        [{_, cached_mod}] ->
-          cached_mod
-
-        [] ->
-          {:ok, mod} = Wasmex.Module.compile(bytes)
-          :ets.insert(:module_cache, {key, mod})
-          mod
-      end
```

Comment:
the nif is holding onto the precompiled module by way of the `ActorReference`, but yeah we don't need to store this stuff in ETS because it's already stored via Rust.